### ISSUE TITLE
A11y accordion

### ISSUE THREAD

**Issue by andresin87** [SUI-Components/SUIComponents#2164]
### [A11Y – Definition](https://www.w3.org/TR/wai-aria-practices/#accordion)
- [x] The title of each accordion header is contained in an element with role [button](https://www.w3.org/TR/wai-aria-1.2/#button).
- [x] Each accordion header button is wrapped in an element with role [heading](https://www.w3.org/TR/wai-aria-1.2/#heading) that has a value set for [aria-level](https://www.w3.org/TR/wai-aria-1.2/#aria-level) that is appropriate for the information architecture of the page.
  - [x] If the native host language has an element with an implicit heading and aria-level, such as an HTML heading tag, a native host language element may be used.
  - [x] The button element is the only element inside the heading element. That is, if there are other visually persistent elements, they are not included inside the heading element.
- [x] If the accordion panel associated with an accordion header is visible, the header button element has [aria-expanded](https://www.w3.org/TR/wai-aria-1.2/#aria-expanded) set to true. If the panel is not visible, [aria-expanded](https://www.w3.org/TR/wai-aria-1.2/#aria-expanded) is set to false.
- [x] The accordion header button element has [aria-controls](https://www.w3.org/TR/wai-aria-1.2/#aria-controls) set to the ID of the element containing the accordion panel content.
- [x] If the accordion panel associated with an accordion header is visible, and if the accordion does not permit the panel to be collapsed, the header button element has [aria-disabled](https://www.w3.org/TR/wai-aria-1.2/#aria-disabled) set to true.
- [x] Optionally, each element that serves as a container for panel content has role [region](https://www.w3.org/TR/wai-aria-1.2/#region) and [aria-labelledby](https://www.w3.org/TR/wai-aria-1.2/#aria-labelledby) with a value that refers to the button that controls display of the panel.
  - [x] Avoid using the region role in circumstances that create landmark region proliferation, e.g., in an accordion that contains more than approximately 6 panels that can be expanded at the same time.
  - [x] Role region is especially helpful to the perception of structure by screen reader users when panels contain heading elements or a nested accordion.

**Comment by turolopezsanabria** [comment]
🌶 súper
